src/application/controllers/property/controller.ts

```diff
@@ -55,7 +53,7 @@ class PropertyController implements IPropertyController {
 
   public async delete(
     req: Request<void, { propertyId: string }>,
-  ): Promise<ResponseDtos.PropertyDto> {
+  ): Promise<undefined> {
```

Comment:
This should be Promise<void>. If you dont specify the return vale thats the default too, so just remove it.